kymatio/frontend/torch_frontend.py

```diff
@@ -17,4 +18,7 @@ def scattering(self, x):
     def forward(self, x):
         """ This function provides the standard calling interface for PyTorch
         modules."""
+
+        input_checks(x)
+
```

Comment:
We sometimes redefine the forward function in torch in some `*D` scatterings, to do size checks and such. This means we will not be doing several input_checks.

Reply:
We do? We should only be redefining `scattering`, if I recall correctly.